misc/config_tools/schema/config.xsd

```diff
@@ -358,8 +358,8 @@ Refer SDM 17.19.2 for details, and use with caution.</xs:documentation>
         <xs:documentation>Specify the Intel Software Guard Extensions (SGX) enclave page cache (EPC) section settings.</xs:documentation>
       </xs:annotation>
     </xs:element>
-    <xs:element name="memory" type="MemoryInfo">
-      <xs:annotation acrn:views="basic, advanced">
+    <xs:element name="memory" type="MemoryInfo" minOccurs="0">
+      <xs:annotation acrn:views="basic, advanced" acrn:applicable-vms="pre-launched, post-launched">
         <xs:documentation>Specify memory information for Service and User VMs.</xs:documentation>
```

Comment:
Change the documentation at line 363 to `<xs:documentation>Specify memory information for User VMs.</xs:documentation>`.  This element doesn't define memory for Service VM.